### PR TITLE
Feature/get service user packages

### PR DESCRIPTION
**Pull request by dunnudge**
## Describe this PR
Get all packages for a service user. Calculations not done but structure of response will stay the same (Unblock FE integration)

### *What changes have we introduced*
- Get packages for service user
- When updating, create core package settings if not found

### *What's missing*
- Calculations (net, gross)
- Tests

### CODE REVIEW

File: LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            // Add core cost
+            var carePackageCostItem = new List<CarePackageCostItemResponse>
+            {
+                new CarePackageCostItemResponse
+                {
+                    Id = package.Id,
+                    Name = package.PackageType.GetDisplayName(),
+                    Type = package.PackageType.GetDisplayName(),
+                    CollectedBy = ClaimCollector.Hackney.GetDisplayName(),
+                    Status = DateTimeOffset.Now.IsInRange(coreCost.StartDate,
```
Comment (brkzkn) [inline LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs:82]: No need to auto calculate for the end. The user is responsible for ending manually.

File: LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+            // Add one off additional needs
+            var oneOffAdditionalNeeds = additionalNeeds.Where(d => d.CostPeriod is PaymentPeriod.OneOff).ToList();
+            carePackageCostItem.AddRange(oneOffAdditionalNeeds.Select(need => new CarePackageCostItemResponse
+            {
+                Id = need.Id,
+                Name = "Additional needs payment / one-off",
+                Type = "Additional Needs One Off",
+                CollectedBy = ClaimCollector.Hackney.GetDisplayName(),
+                Status = DateTimeOffset.Now.IsInRange(need.StartDate, need.EndDate ?? DateTimeOffset.Now.AddYears(10)) ? "Active" : "Ended",
```
Comment (brkzkn) [inline LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs:100]: Same for above

File: LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+            // Add weekly additional needs
+            var weeklyAdditionalNeeds = additionalNeeds.Where(d => d.CostPeriod is PaymentPeriod.Weekly).ToList();
+            carePackageCostItem.AddRange(weeklyAdditionalNeeds.Select(need => new CarePackageCostItemResponse
+            {
+                Id = need.Id,
+                Name = "Additional needs payment / wk",
+                Type = "Additional Needs Weekly",
+                CollectedBy = ClaimCollector.Hackney.GetDisplayName(),
+                Status = DateTimeOffset.Now.IsInRange(need.StartDate, need.EndDate ?? DateTimeOffset.Now.AddYears(10)) ? "Active" : "Ended",
```
Comment (brkzkn) [inline LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs:114]: Same for above

File: LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs (hunk 1 above)
Comment (dunnudge) [reply LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs:82]: @brkzkn opinion

Comment (brkzkn) [reply LBH.AdultSocialCare.Api/V1/UseCase/CarePackages/Concrete/GetServiceUserPackagesUseCase.cs:82]: I got your point Duncan. 

CoreElement status should be the same as Package status.
For ANPs, we should keep status column. Status might be `Active, End, Cancelled`. Active, End status comes from Package status. When the user remove ANPs, status will be `Cancelled`.

**Pull request merged**